danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/GitLabKtx.kt

```diff
@@ -0,0 +1,47 @@
+package systems.danger.kotlin
+
+// extensions on entities from [GitLab] file
+
+/**
+ * Changed lines in this MR
+ */
+val GitLabMergeRequest.changedLines: MergeRequestChangedLines
+    get() {
+        val commandRawOutput = Utils().exec("git diff --numstat ${diffRefs.startSha} ${diffRefs.headSha}")
```

Comment:
I'm not sure what is the best way how to use `Utils` sine it's a class and not an `object`. Creating new instance here seems wrong.

Reply:
I agree, it feels wrong, the correct way to access utils should be from the DangerDSL, but to access that you need to be in the Dangerfile.
Then what I would do is create a simple class that can run commands (example from danger swift https://github.com/danger/swift/blob/master/Sources/DangerShellExecutor/ShellExecutor.swift), and use that both in utils and here.

Maybe this could also help us to find a way to test this?

Reply:
agree as well, maybe convert Utils to an `object` ?

Reply:
I've left this unprocessed for now and I would focus on that in the next MR

Reply:
We could make (I’m fine to see it next though) a Cmd class like the one on the runner that we could share wherever we have to execute shell code

Reply:
Ok, I've prepared something, I will create a PR once this one is resolved :) [Here](https://github.com/davidbilik/kotlin/commit/0dca794954fab43b58d74eb6c513dfb50d6c46f2) is a commit in my fork